gap/perm.gi

```diff
@@ -16,19 +16,19 @@
 ##
 #############################################################################
 
-#! @BeginChunk VeryFewPoints
-#! If a permutation group acts only on a few points (the current limit is at
-#! most 10 points) then a stabiliser chain is computed by the randomized &GAP;
-#! library function for that purpose. If the method is successful then the
+#! @BeginChunk VerySmallPoints
+#! If a permutation group moves only very small points
+#! (currently, this means that its largest moved point is at most 10),
+#! then a stabiliser chain is computed by the randomized &GAP;
+#! library function for that purpose. If the method is successful, then the
 #! calling node becomes a leaf node in the recursive scheme. If the input group
-#! acts on more than 10 points then the method returns <C>NeverApplicable</C>.
+#! acts on more than 10 points, then the method returns <C>NeverApplicable</C>.
```

Comment:
I realize you only edited the text in a minor way. But perhaps we could also expand the explanation a bit: I think the reason for the existence of this method is that when dealing with very small degree permutation groups, there is little point in "reognition", as just computing a stab chain is faster than anything fancy we might do.

So perhaps we could mention this briefly?

Of course that's true if all we care about is "some" recognition tree. If we want to do more serious work with it, we may want to really "recognize" every part explicitly... Whether this means this method should be disabled in the future, or whether instead one should explicitly rewrite the initial recognition tree into a more cleverly arrange composition tree (as done in Magma) is another question.

Anyhow: I am also fine with merging this PR as-is!

Reply:
I had a go.